test/recipes.spec.js

```diff
@@ -140,5 +143,66 @@ describe('Recipes', () => {
           done(err);
         });
     });
+
+    it('should update recipe', done => {
+      chai
+        .request(server)
+        .put(`${baseUrl}/${slug}`)
+        .set({ authorization: token })
+        .send({ preparationTime: 5000 })
+        .end((err, res) => {
+          const { body } = res;
+          expect(res).to.have.status(200);
+          expect(body.recipe).to.be.an('object');
+          expect(body.recipe.id).to.be.a('number');
+          expect(body.recipe).to.haveOwnProperty('title');
+          done(err);
+        });
+    });
+
+    it('should return an error if user does not exist', done => {
+      chai
+        .request(server)
+        .put(`${baseUrl}/${slug}`)
+        .set({ authorization: generateToken({ id: 1000 }) })
+        .send({ preparationTime: 5000 })
+        .end((err, res) => {
+          expect(res).to.have.status(404);
+          done(err);
+        });
+    });
+
+    it('should return an error if wrong input string', done => {
+      chai
+        .request(server)
+        .put(`${baseUrl}/${slug}`)
+        .set({ authorization: token })
+        .send({ preparationTime: 'time' })
+        .end((err, res) => {
+          expect(res).to.have.status(500);
+          done(err);
+        });
+    });
+
+    it('should delete recipe', done => {
+      chai
+        .request(server)
+        .delete(`${baseUrl}/${slug}`)
+        .set({ authorization: token })
+        .end((err, res) => {
+          expect(res).to.have.status(200);
+          done(err);
+        });
+    });
+    it('should return an error if user does not exist', done => {
+      chai
+        .request(server)
+        .delete(`${baseUrl}/${slug}`)
+        .set({ authorization: generateToken({ id: 1000 }) })
+        .end((err, res) => {
+          expect(res).to.have.status(404);
+          done(err);
+        });
+    });
```

Comment:
it would be good to also add a test for if the recipe doesn't exist when deleting

Reply:
noted. I will include the test

Reply:
done